tox.ini

```diff
@@ -31,7 +31,7 @@ commands = mypy -p snorkel
 description = run coverage checks
 basepython = python3
 usedevelop = True
-commands = pytest --cov=snorkel test/
+commands = python -m pytest --cov=snorkel test/
```

Comment:
Why was this necessary to add?

Reply:
It wasn't strictly but just standardized the pytest call (see the py env above) when debugging the PyArrow+PyTorch issue